### ISSUE TITLE
Can only run the app from the root

### ISSUE THREAD

**Issue by classikd** [jorgegorka/svelte-router#5]
Hello,

I encounter a problem when I load or refresh my app from anywhere, except from http://localhost:5000/.
For example, you can see what happens when I refresh the page _admin/roles/index_:
![error](https://user-images.githubusercontent.com/5224312/63472854-b6488580-c473-11e9-9a3e-1db775b5616f.JPG)

Do you have any idea of what can cause that ?
Thank you

**Comment by jorgegorka** [comment]
Hello @classikd 

It looks like you have a syntax error in your code. _Unexpected token <_

**Comment by classikd** [comment]
I didn't have a syntax error. 
I solved the problem by adding `/` to paths in public/index.html
![index-path](https://user-images.githubusercontent.com/5224312/63545658-3e886280-c528-11e9-8d46-3ed44402673b.JPG)

I installed the default svelte template like 2 months ago but a recent [commit ](https://github.com/sveltejs/template/commit/710c36b036f98262279d22f638940746ce7ab4f5#diff-528c3923d718a8860f5d8c05c3931c55) changed those paths.

I didn't have this problem with [svelte-router-spa](https://github.com/ItalyPaleAle/svelte-spa-router) tha's why I came here. 
But everything it's ok now :)
Thanks for your work by the way.
